EIPS/eip-zodiac.md

```diff
@@ -0,0 +1,175 @@
+---
+eip: <to be assigned>
+title: Zodiac
+description: A composable design philosophy for DAOs.
+author: Auryn Macmillan (@auryn-macmillan), Kei Kreutler (@keikreutler)
+discussions-to: https://gnosisguild.mirror.xyz/OuhG5s2X5uSVBx1EK4tKPhnUc91Wh9YM0fwSnC8UNcg
```

Comment:
`discussions-to` should be a link to https://ethereum-magicians.org/

Reply:
Spinning up a thread now.